provider/pkg/provider/local/commandController.go

```diff
@@ -31,7 +31,8 @@ var _ = (infer.ExplicitDependencies[CommandInputs, CommandOutputs])((*Command)(n
 
 // This is the Create method. This will be run on every Command resource creation.
 func (c *Command) Create(ctx p.Context, name string, input CommandInputs, preview bool) (string, CommandOutputs, error) {
-	state := CommandOutputs{CommandInputs: input}
+	outputs := BaseOutputs{Stdout: *input.Stdin, Stderr: *input.Stdin}
```

Comment:
I don't understand why we are setting `stderr` to `*input.stdin`. `stderr` should not be set until a command is actually run.